### ISSUE TITLE
webcrank-0.2 can't build its own test suite

### ISSUE THREAD

**Issue by peti** [webcrank/webcrank.hs#15]
Citing from http://hydra.cryp.to/build/785343/nixlog/2/raw:

```
Preprocessing test suite 'tests' for webcrank-0.2...

test/tests.hs:3:8:
    Could not find module ‘DecisionTests’
    Use -v to see a list of the files searched for.

test/tests.hs:4:8:
    Could not find module ‘ParserTests’
    Use -v to see a list of the files searched for.

test/tests.hs:5:8:
    Could not find module ‘HandleRequestTests’
    Use -v to see a list of the files searched for.
```

**Comment by purefn** [comment]
Strange, thanks for reporting. This is usually because the files aren't included in the source distribution, isn't it?

**Comment by peti** [comment]
Yes. I guess they aren't listed in the Cabal file.

**Comment by purefn** [comment]
Just uploaded a fixed 0.2.0.1 release to hackage.  Thanks again for letting me know.
